ui/src/tasks/reducers/tasks.test.ts

```diff
@@ -5,7 +5,8 @@ import tasksReducer, {
 import {setTaskOption} from 'src/tasks/actions'
 import {TaskSchedule} from 'src/utils/taskOptionsToFluxScript'
 
-describe('tasksReducer', () => {
+// skipping this test since the cron and interval values should not be reset when toggling between schedule tasks
+describe.skip('tasksReducer', () => {
```

Comment:
if this asserts something that is no longer the case, or it represents outdated assumptions, i'd say just remove the whole assertion. i think keeping it around will be more confusing than helpful. i appreciate the hesitation to delete an entire file someone else wrote though :)

if we're not sure what this is supposed to do, let's get some clarification from russ.

what do you think?

Reply:
I appreciate the sympathy - definitely didn't wanna go around deleting files and taking names my second week, but it doesn't seem applicable. The only reason why I'd want to keep it would be to have a basis for fleshing out tests on the reducer later on. I'll go ahead and fix the tests so that they're consistent with the new logic of not overwriting the previous values